tests/utils/configmaps.go

```diff
@@ -81,6 +81,35 @@ func CopyConfigMap(client kubernetes.Interface, srcNamespace, srcName, destNames
 	return destName, nil
 }
 
+// CreateCertConfigMapWeirdFilename copies a configmap with a different key value
+func CreateCertConfigMapWeirdFilename(client kubernetes.Interface, destNamespace, srcNamespace string) (string, error) {
+	var certBytes string
+	srcName := FileHostCertConfigMap
+	srcCm, err := client.CoreV1().ConfigMaps(srcNamespace).Get(context.TODO(), srcName, metav1.GetOptions{})
+	if err != nil {
+		return "", err
+	}
+
+	for _, value := range srcCm.Data {
+		certBytes = value
+	}
```

Comment:
Why the iteration? It gets the last item, so let's just to that.

Reply:
I wasn't sure about the idiomatic way to do it, there's usually only going to be one element here, I could add a break for clarity.